include/seastar/core/memory.hh

```diff
@@ -157,6 +157,13 @@ void configure(std::vector<resource::memory> m, bool mbind,
  */
 void enable_abort_on_allocation_failure(bool enabled = true);
 
+/**
+ * Return true if the global abort on allocation failure behavior
+ * is enabled, or false otherwise. Always returns false if the 
+ * default (system) allocator is being used.
+ */
+bool is_abort_on_allocation_failure();
+
```

Comment:
could you please add a few words on the commit message about the motivation for this one? 

I am trying to understand why an application needs it. If it needs to enable, it will enable regardless of the previous value.

Reply:
Thanks for the feedback @raphaelsc. I've updated the commit comment to the following:

```
Allow application to query the abort on OOM state

Add an is_abort_on_allocation_failure method which returns
the global state of this reactor option.

This is useful for applications that which to query this state in order
to expose it for debugging or administration purposes. Applications
could try to track this themselves, but this is unreliable since the
reactor itself can change this value based on the command line flags.
```

Let me know if you think it is sufficient.